meilisearch/models/task.py

```diff
@@ -3,11 +3,14 @@
 from datetime import datetime
 from typing import Any, Dict, List, Optional, Union
 
+import pydantic
 from camel_converter.pydantic_base import CamelBase
 
+from meilisearch._utils import is_pydantic_2, iso_to_date_time
+
 
 class Task(CamelBase):
-    uid: str
+    uid: int
```

Comment:
Is this not breaking?

Reply:
This is where my stray `print` came from with testing this out. It was erroring so I wanted to see the values. `uid` gets returned from Meilisearch like: `{"uid": 123}` and not `{"uid": "123"}`. Pydantic used to convert the int into a string, but now considers this an error since it isn't the correct type.

Reply:
Indeed the uid is an `int`. I was surprised we let it as `str` in the code. but I wonder if it will break something

Reply:
Yes, that's exactly what I meant, anyway the task_uid is well-defined as an `int` so I think it's better to change it.